### PR TITLE
Add fixed width functionality to option parameters

### PR DESCRIPTION
**Pull request by dmoore1989**
Users of this application can also set a fixed image width instead of setting a step distance.  The fixed width overrides step parameters/default values and sets an average step distance for each point .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ parameter | default | unit | purpose
 `height` | `30` | pixels | height of image
 `line_color` | `4A8FED` | hex | color of line
 `step` | `30` | pixels | distance between steps
+`width` | | pixels | total graph (overides step distance)
```
Comment (afeld) [inline README.md]: Mind just adding the word "width" to the `purpose`?

File: server.rb [hunk 2]
```diff
@@ -7,6 +7,14 @@ def param_with_default(key, default)
   params[key] ? params[key] : default
 end
 
+def width_with_default( val_length)
```
Comment (afeld) [inline server.rb]: Minor: mind removing the space after the first parenthesis?

**Comment by afeld** [comment]
This is _awesome_, thanks so much! Just a couple minor things. Leave a comment when it's ready and we'll get this thing merged!

**Comment by dmoore1989** [comment]
Should be good to go!

**Comment by afeld** [comment]
Hooray! Thanks a ton.

**Pull request merged**